### PR TITLE
Added youtube videos and app stuff to seismic gpg. Improved survey section

### PR DESCRIPTION
**Pull request by Pbellive**
…ction.

### CODE REVIEW

File: content/seismic/raw_refraction_data.html [hunk 1]
```diff
@@ -38,13 +38,13 @@
   <tbody>
     <tr>
         <td valign="top" valign="top" style="width:30%;">
-    <input name="radiobutton" type="radio" onClick="MM_swapImage('firstbreaks','','./../../../_images/firstbreak1.gif',1)" value="radiobutton" checked>
+    <input name="radiobutton" type="radio" onClick="MM_swapImage('firstbreaks','','./images/firstbreak1.gif',1)" value="radiobutton" checked>
```
Comment (lheagy) [inline content/seismic/raw_refraction_data.html]: again, these need to be reverted @Pbellive

File: content/seismic/waves_at_interfaces.rst [hunk 2]
```diff
@@ -28,11 +33,19 @@ Displacement of the earth from equilibrium position must be continuous across an
...
+.. raw:: html
+
+    <div style="margin-top:10px; margin-bottom:20px;" align="center">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/9OpL3OFuVXo" frameborder="0" allowfullscreen></iframe allowfullscreen>
+    </iframe>
+    </div>
 
+You can explore the concept further using an interactive web app `here <https://phet.colorado.edu/sims/html/wave-on-a-string/latest/wave-on-a-string_en.html>`__.
+    
 **Remark:**  Large amplitudes of the transmitted wave are sometimes counter-
```
Comment (lheagy) [inline content/seismic/waves_at_interfaces.rst]: here, you can use a `.. note::` environment to have this highlighted eg. 

image

File: content/seismic/waves_at_interfaces.rst [hunk 3]
```diff
@@ -106,10 +127,32 @@ a "head wave". It is somewhat analogous to the bow wave of a moving boat. This i
 .. figure:: ./images/criticalrefraction.gif
 	:align: center
 
-The animation of wavefront propagation shown earlier (and directly below) is again useful here. The head wavefront is clearly visible after the initial downgoing wave from the source hits the interface.
+The following video from `the IRIS group <https://www.iris.edu/hq/programs/epo>`__ illustrates the propagation of reflected, sub-critically refracted and critically refracted rays.
+
+.. raw:: html
+
+    <div style="margin-top:10px; margin-bottom:20px;" align="center">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/FygYDmm99SA" frameborder="0" allowfullscreen></iframe>
```
Comment (lheagy) [inline content/seismic/waves_at_interfaces.rst]: can you please add the `?rel=0` to the end of the url here?

File: content/seismic/raw_refraction_data.html [hunk 4]
```diff
@@ -29,7 +29,7 @@
 pWin=window.open(urlstring,winName, "toolbar=0,scrollbars=1,directories=0,resizable=1,height=" + height + ",width=" + width + ",status=0"); 
 }
 
-MM_preloadImages('./../../_images/firstbreak2.gif','./../../_images/firstbreak3.gif','./../../_images/firstbreak1.gif');
+MM_preloadImages('./images/firstbreak2.gif','./images/firstbreak3.gif','./images/firstbreak1.gif');
```
Comment (Pbellive) [inline content/seismic/raw_refraction_data.html]: My understanding is that the figure using that radio button should be removed completely. That was my reasoning for leaving #112  open. This issue and that one can be closed when I update http://gpg.geosci.xyz/content/seismic/seismic_data.html

**Pull request merged**